lib/query_tracker.rb

```diff
@@ -17,8 +17,10 @@ def self.track
         tables = PgQuery.parse(sql).tables.map(&:to_sym)
 
         root = Rails.root.to_s
-        vendor = Rails.root.join('vendor', 'bundle').to_s
-        location = caller.find { |line| line.include?(root) && !line.include?(vendor) }
+        vendor = ENV['GEM_PATH'].split(':')
+        location = caller.find do |line|
+          line.include?(root) && !vendor.any? { |v| line.include?(v) }
+        end
```

Comment:
I think we can use the `Gem` API for this? and switch from `!any?` to `none?`

```suggestion
        location = caller.find do |line|
          line.include?(root) && Gem.path.none? { |v| line.include?(v) }
        end
```

Reply:
Thanks, TIL 🙂